### PR TITLE
Be also a maintainer for hpp-fcl to maintain v1 abi branch for hpp-xxx stack

### PR DESCRIPTION
**Pull request by ymontmarin**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ymontmarin** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml
Comment (jcarpent) [inline conda-forge.yml:2]: What does it do?

**Comment by ymontmarin** [comment]
Hi ! I dit not bump the build number as my change are only meta !

I would like to also be a maintainer of this recipe as I am maintainer of other `hpp-` packages following the graph dependency presented in [this slide](https://docs.google.com/presentation/d/1c6P_8dOgEIsKOxhLtcubc9fAypdmoZjA326JkjqC7lA/edit#slide=id.g13a30079a15_0_0).

As mentioned in https://github.com/conda-forge/hpp-pinocchio-feedstock/pull/13 , `hpp-pinocchio` required `pinocchio 2.x.x` and `hpp-fcl 1.x.x`. A solution I first tried was to freeze all the dependency packages at time of `v1.8.1`. But it is not in the philosophy of “just use conda-forge” and it takes us away from a simple migration when pinocchio 3 will be available.

As suggested by @traverso, a more elegant way is to maintain a `v1` for `hpp-fcl`. I mentioned in this PR this branch in `abi_migration_branches`, and as a maintainer I will be able to create and maintain the `v1` branch as in my [fork](https://github.com/ymontmarin/hpp-fcl-feedstock/tree/v1).

Concretely it will maintain the 1.8.1 with new version of boost / assimp / urdfdom... And we will not need to freeze everything but only hpp-fcl for the hpp-stack !

**Comment by ymontmarin** [comment]
@traversaro, after this PR is merged (so v1 is added as abi_migration_branch in `conda-forge.yml`) and that I successfully created the v1 branch as a maintainer. How do I effectively rebuild hpp-fcl with the new migrations (boost, assimp, urdfdom) of https://github.com/conda-forge/conda-forge-pinning-feedstock/tree/main/recipe/migrations .

Do I only need to add the corresponding migration files in the recipe folder in the v1 branch ?

Thank you !

File: conda-forge.yml
Comment (ymontmarin) [reply conda-forge.yml:2]: I explain everything in my comment

**Comment by traversaro** [comment]
> @traversaro, after this PR is merged (so v1 is added as abi_migration_branch in `conda-forge.yml`) and that I successfully created the v1 branch as a maintainer. How do I effectively rebuild hpp-fcl with the new migrations (boost, assimp, urdfdom) of https://github.com/conda-forge/conda-forge-pinning-feedstock/tree/main/recipe/migrations .
> 
> Do I only need to add the corresponding migration files in the recipe folder in the v1 branch ?
> 
> Thank you !

No, in theory migrations PRs will be opened by the autotick-bot .

**Review by jcarpent — APPROVED** [review]

**Comment by ymontmarin** [comment]
@traversaro Ok thank you ! Everything is ready now ! Do you know when the bot will trigger the updates ? Will a manual PR with the 3 migration files added do the trick ?

**Pull request merged**
